modules/ossm-routing.adoc

```diff
@@ -13,21 +13,23 @@ Configure your service mesh by adding your own traffic configuration to {Product
 
 You can route requests dynamically to multiple versions of a microservice through {ProductName} with a virtual service. With virtual services, you can:
 
-* Address multiple application services through a single virtual service. If your mesh uses Kubernetes, for example, you can configure a virtual service to handle all services in a specific namespace. Mapping a single virtual service to many services is particularly useful in facilitating turning a monolithic application into a composite service built out of distinct microservices without requiring the consumers of the service to adapt to the transition.
+* Address multiple application services through a single virtual service. If your mesh uses Kubernetes, for example, you can configure a virtual service to handle all services in a specific namespace. Virtual service allows you to turn a monolithic application into a service comprised of distinct microservices with a seamless consumer experience. 
```

Comment:
I think this should be either:

A virtual service allows you to turn...

or:

Virtual services allow you to turn....